packages/reakit-playground/src/PlaygroundPreview.tsx

```diff
@@ -101,6 +101,11 @@ export function PlaygroundPreview({
     unmount
   ]);
 
+  React.useEffect(() => {
+    // Ensure that we unmount the React component when the effect is destroyed.
+    return () => unmount();
+  });
```

Comment:
```suggestion
  }, [unmount]);
```

Otherwise it will trigger on every render.

Reply:
Good catch. 🙈